book/security.md

```diff
@@ -868,7 +868,15 @@ class JSContext:
 The `request` function can now check the `top_level_url` argument
 before sending `SameSite` cookies. Remember that `SameSite` cookies
 are only sent for `GET` requests or if the new URL and the top-level
-URL have the same host name:
+URL have the same host name:[^schemeful]
+
+[^schemeful]: At I write this, some browsers also check that the new
```

Comment:
Does look like it shipped in M89: https://chromestatus.com/feature/5096179480133632